integration_tests/dbt_project.yml

```diff
@@ -1,44 +1,49 @@
+version: '0.4.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
   twitter_organic_source:
-    twitter_organic_schema: social_media_rollup_integration_tests
-    twitter_organic_account_history_identifer: "twitter_organic_account_history_data"
-    twitter_organic_organic_tweet_report_identifer: "twitter_organic_organic_tweet_report_data"
-    twitter_organic_tweet_identifer: "twitter_organic_tweet_data"
-    twitter_organic_twitter_user_history_identifer: "twitter_organic_twitter_user_history_data"
+    twitter_organic_schema: social_media_rollup_integration_tests_1
+    twitter_account_history_identifier: "account_history"
+    twitter_organic_tweet_report_identifier: "organic_tweet_report"
+    twitter_tweet_identifier: "tweet"
+    twitter_twitter_user_history_identifier: "twitter_user_history"
   linkedin_pages_source:
-    linkedin_pages_schema: social_media_rollup_integration_tests
+    linkedin_pages_schema: social_media_rollup_integration_tests_1
     linkedin_pages_organization_identifier: "linkedin_pages_organization_data"
     linkedin_pages_organization_ugc_post_identifier: "linkedin_pages_organization_ugc_post_data"
     linkedin_pages_share_statistic_identifier: "linkedin_pages_share_statistic_data"
     linkedin_pages_post_content_identifier: "linkedin_pages_post_content_data"
     linkedin_pages_ugc_post_history_identifier: "linkedin_pages_ugc_post_history_data"
     linkedin_pages_ugc_post_share_statistic_identifier: "linkedin_pages_ugc_post_share_statistic_data"
   instagram_business_source:
-    instagram_business_schema: social_media_rollup_integration_tests
+    instagram_business_schema: social_media_rollup_integration_tests_1
     instagram_business_media_history_identifier: "instagram_business_media_history_data"
     instagram_business_media_insights_identifier: "instagram_business_media_insights_data"
     instagram_business_user_history_identifier: "instagram_business_user_history_data"
   facebook_pages_source:
-    facebook_pages_schema: social_media_rollup_integration_tests
+    facebook_pages_schema: social_media_rollup_integration_tests_1
     facebook_pages_daily_page_metrics_total_identifier: "facebook_pages_daily_page_metrics_total_data"
     facebook_pages_lifetime_post_metrics_total_identifier: "facebook_pages_lifetime_post_metrics_total_data"
     facebook_pages_page_identifier: "facebook_pages_page_data"
     facebook_pages_post_history_identifier: "facebook_pages_post_history_data"
 
+  ## Used for local validation testing
+  # twitter_organic_schema: <local_schema_name>
+  # linkedin_pages_schema: <local_schema_name>
+  # instagram_business_schema: <local_schema_name>
+  # facebook_pages_schema: <local_schema_name>
+
 dispatch:
   - macro_namespace: dbt_utils
     search_order: ['spark_utils', 'dbt_utils']
 
 models: 
+  +schema: "social_media_{{ var('directed_schema','dev') }}"
   social_media_reporting:
     +materialized: table
-  twitter_organic_source:
-    +materialized: table
```

Comment:
Same note from the transform PR. This was masking the window function issues on Redshift. I have not turned this off on the other packages because the fix has not yet been applied. Ideally these table materialization configs can all be removed in the future.